src/server.jl

```diff
@@ -370,8 +355,9 @@ function ws_tracker(ws::HTTP.WebSockets.WebSocket, target::AbstractString)
         # forces the websocket to stay open until it's closed by LiveServer (and
         # not by the browser) upon writing a `update` message on the websocket.
         # See update_and_close_viewers
-        while isopen(ws.io)
-            sleep(0.1)
+        while !WebSockets.isclosed(ws)
```

Comment:
This seems IMO the more appropriate way to avoid idle connections getting dropped. The websockets protocol specifically lists unidirectional `PONG` messages as a way for one side to provide a "heartbeat" to a remote to avoid closure due to idleness.